[2023-01-05 00:52:39,485] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: stock_data_pipeline_dag.delete_cluster_dataproc_task scheduled__2023-01-04T00:00:00+00:00 [queued]>
[2023-01-05 00:52:39,494] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: stock_data_pipeline_dag.delete_cluster_dataproc_task scheduled__2023-01-04T00:00:00+00:00 [queued]>
[2023-01-05 00:52:39,494] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-01-05 00:52:39,494] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-01-05 00:52:39,494] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-01-05 00:52:39,507] {taskinstance.py:1377} INFO - Executing <Task(DataprocDeleteClusterOperator): delete_cluster_dataproc_task> on 2023-01-04 00:00:00+00:00
[2023-01-05 00:52:39,517] {standard_task_runner.py:52} INFO - Started process 3948 to run task
[2023-01-05 00:52:39,526] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'stock_data_pipeline_dag', 'delete_cluster_dataproc_task', 'scheduled__2023-01-04T00:00:00+00:00', '--job-id', '39', '--raw', '--subdir', 'DAGS_FOLDER/stock_data_pipeline_dag.py', '--cfg-path', '/tmp/tmpqrk6eszn', '--error-file', '/tmp/tmpbafy0ltk']
[2023-01-05 00:52:39,527] {standard_task_runner.py:80} INFO - Job 39: Subtask delete_cluster_dataproc_task
[2023-01-05 00:52:39,592] {task_command.py:370} INFO - Running <TaskInstance: stock_data_pipeline_dag.delete_cluster_dataproc_task scheduled__2023-01-04T00:00:00+00:00 [running]> on host 2302f2c95f83
[2023-01-05 00:52:39,661] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=stock_data_pipeline_dag
AIRFLOW_CTX_TASK_ID=delete_cluster_dataproc_task
AIRFLOW_CTX_EXECUTION_DATE=2023-01-04T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-04T00:00:00+00:00
[2023-01-05 00:52:39,664] {base.py:68} INFO - Using connection ID 'google_cloud_default' for task execution.
[2023-01-05 00:52:39,664] {dataproc.py:816} INFO - Deleting cluster: move-data
[2023-01-05 00:53:26,945] {dataproc.py:828} INFO - Cluster deleted.
[2023-01-05 00:53:26,974] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=stock_data_pipeline_dag, task_id=delete_cluster_dataproc_task, execution_date=20230104T000000, start_date=20230105T005239, end_date=20230105T005326
[2023-01-05 00:53:27,019] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-01-05 00:53:27,062] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
